smac/optimizer/smbo.py

```diff
@@ -99,11 +101,25 @@ def run(self):
             The best found configuration
         '''
         self.stats.start_timing()
-        try:
-            self.incumbent = self.initial_design.run()
-        except FirstRunCrashedException as err:
-            if self.scenario.abort_on_first_run_crash:
-                raise
+        # Initialization, depends on input
```

Comment:
can we please move Lines 104-122 in function?
(otherwise we would need to copy the code for new optimizers besides SMBO.